### PR TITLE
Improve TOCO output

### PR DESCRIPTION
**Pull request by benjamintanweihao**
Currently the output is a binary string contain `\n` characters, making the output not very readable. This change makes renders the output as the user expects.

### CODE REVIEW

**Review by rmlarsen — APPROVED** [review]

File: tensorflow/contrib/lite/python/convert.py [hunk 1]
```diff
@@ -131,8 +131,8 @@ def toco_convert_protos(model_flags_str, toco_flags_str, input_data_str):
       stuff = fp_output.read()
       return stuff
     else:
-      raise RuntimeError("TOCO failed see console for info.\n%s\n%s\n" %
-                         (stdout, stderr))
+      raise RuntimeError("TOCO failed. See console for info.\n%s\n%s\n" %
```
Comment (rmlarsen) [inline tensorflow/contrib/lite/python/convert.py]: Could you add a unit test for this?

**Review by rmlarsen — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/lite/python/convert.py (hunk 1 above)
Comment (benjamintanweihao) [reply tensorflow/contrib/lite/python/convert.py]: Will do! Give me a bit of time to work on it :+1:

**Comment by benjamintanweihao** [comment]
Yup this is completely my bad. I’ll fix it.

File: tensorflow/contrib/lite/python/convert.py (hunk 1 above)
Comment (benjamintanweihao) [reply tensorflow/contrib/lite/python/convert.py]: @rmlarsen Done!

**Comment by tensorflowbutler** [comment]
Nagging Assignee @rmlarsen: It has been 19 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by benjamintanweihao** [comment]
Will do it tomorrow. :)

**Comment by rmlarsen** [comment]
@benjamintanweihao are you planning to move forward with this?

**Comment by rmlarsen** [comment]
Thanks. Please resolve the conflicts.

**Comment by benjamintanweihao** [comment]
Argh. I will redo this.

**Pull request closed without merging**